docs/models/index.md

```diff
@@ -348,69 +342,59 @@ If you try to do so the `ModelError` will be raised.
 
-On the contrary in `pydantic` the default option is to ignore such extra fields, therefore `ormar` provides an `Meta.extra` setting to behave in the same way.
+On the contrary in `pydantic` the default option is to ignore such extra fields, therefore `ormar` provides an `ormar_config.extra` setting to behave in the same way.
 
 To ignore extra fields passed to `ormar` set this setting to `Extra.ignore` instead of default `Extra.forbid`.
 
 Note that `ormar` does not allow accepting extra fields, you can only ignore them or forbid them (raise exception if present)
 
 ```python
-from ormar import Extra
+from ormar import Extra, OrmarConfig
 
 class Child(ormar.Model):
-    class Meta(ormar.ModelMeta):
-        tablename = "children"
-        metadata = metadata
-        database = database
-        extra = Extra.ignore  # set extra setting to prevent exceptions on extra fields presence
+    ormar_config = OrmarConfig(
+        tablename="children",
+        extra=Extra.ignore  # set extra setting to prevent exceptions on extra fields presence
+    )
 
     id: int = ormar.Integer(name="child_id", primary_key=True)
     first_name: str = ormar.String(name="fname", max_length=100)
     last_name: str = ormar.String(name="lname", max_length=100)
 ```
 
-To set the same setting on all model check the [best practices]("../models/index/#best-practice") and `BaseMeta` concept.
+To set the same setting on all model check the [best practices]("../models/index/#best-practice") and `base_ormar_config` concept.
 
 ## Model sort order
 
 When querying the database with given model by default the Model is ordered by the `primary_key`
 column ascending. If you wish to change the default behaviour you can do it by providing `orders_by`
-parameter to model `Meta` class.
+parameter to model `ormar_config` object.
 
 Sample default ordering:
 ```python
-database = databases.Database(DATABASE_URL)
-metadata = sqlalchemy.MetaData()
-
+base_ormar_config = ormar.OrmarConfig(
+    database=databases.Database(DATABASE_URL),
+    metadata=sqlalchemy.MetaData(),
+)
 
-class BaseMeta(ormar.ModelMeta):
-    metadata = metadata
-    database = database
 
 # default sort by column id ascending
 class Author(ormar.Model):
-    class Meta(BaseMeta):
-        tablename = "authors"
+    ormar_config = base_ormar_config.copy()
 
     id: int = ormar.Integer(primary_key=True)
     name: str = ormar.String(max_length=100)
 ```
 Modified
 ```python
-
-database = databases.Database(DATABASE_URL)
-metadata = sqlalchemy.MetaData()
-
-
-class BaseMeta(ormar.ModelMeta):
-    metadata = metadata
-    database = database
+base_ormar_config = ormar.OrmarConfig(
+    database=databases.Database(DATABASE_URL),
+    metadata=sqlalchemy.MetaData(),
+)
 
 # now default sort by name descending
 class Author(ormar.Model):
-    class Meta(BaseMeta):
-        tablename = "authors"
-        orders_by = ["-name"]
+    ormar_config = base_ormar_config.copy(orders_by = ["-name"])
```

Comment:
would add back explicit table name as this is one of the first section users read.